src/functional/predicates.h

```diff
@@ -39,6 +39,13 @@ struct BinaryFunctor {
   }
 };
 
+/**
+ * Macro to set up unary-functions from marian::functional::Ops.
+ *
```

Comment:
I guess we can remove this empty line to save vertical space. This will not change the format of the doxygen doc as we use auto brief.

Reply:
I've removed empty lines